salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -41,6 +41,13 @@ Skip node {{ node }}, already in {{ node_version }} newer than {{ dest_version }
 
   {%- else %}
 
+Wait for API server to be available:
```

Comment:
That will not work as you are in a `for` two state will have the same name you need to add the node name here.
Maybe `Wait for API server to be available for {{ node }}`

Reply:
`on`, not `for`.

Reply:
Hmm it's not really `on` because we not check API server on this node but we check that API server available to be able to upgrade the node

Reply:
Oh but yes we wait **on** this node so `on` maybe :stuck_out_tongue: